### PR TITLE
Adding the CORE API to the Science section

### PR DESCRIPTION
**Pull request by mcancellieri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,6 +387,7 @@ API | Description | Auth | HTTPS | Link |
 API | Description | Auth | HTTPS | Link |
 |---|---|---|---|---|
 | arcsecond.io | Multiple astronomy data sources | No | Yes | [Go!](https://api.arcsecond.io/) |
+|CORE| Access the world's Open Access research papers|  `apiKey`|Yes|[Go!](https://core.ac.uk/services#api) |
```
Comment (jbrooksuk) [inline README.md]: Needs spaces between the pipes please!

**Comment by jbrooksuk** [comment]
You're the man!

**Comment by jbrooksuk** [comment]
And a uke player too?

**Comment by mcancellieri** [comment]
Ahhahah yes! I try really hard to be a uke player, not sure about the results though :)

**Comment by jbrooksuk** [comment]
Me too 🙈

**Pull request merged**